Lombiq.NodeJs.Extensions.Samples/Lombiq.NodeJs.Extensions.Samples.csproj

```diff
@@ -14,12 +14,16 @@
     <Folder Include="wwwwNonDefaultCssFolder\" />
   </ItemGroup>
 
-  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Lombiq.Npm.Targets\Lombiq.Npm.Targets.props" />
-  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\Lombiq.Npm.Targets\Lombiq.Npm.Targets.targets" />
-  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.targets" />
+  <Import Project="..\..\Lombiq.Npm.Targets\Lombiq.Npm.Targets.props" />
+  <Import Project="..\..\Lombiq.Npm.Targets\Lombiq.Npm.Targets.targets" />
+  <Import Project="..\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.targets" />
 
-  <ItemGroup Condition="'$(NuGetBuild)' != 'true'">
+  <ItemGroup>
     <ProjectReference Include="..\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <NpmDotnetPrebuildWatchedFiles Include="package.json;Assets/**/*.*;NonDefaultScssFolder/**/*.*"/>
```

Comment:
It's necessary so that changes in the _NonDefaultScssFolder_ will trigger a re-build of this modules assets. It's only necessary in projects that use non-default paths.

Reply:
I'll add it to the documentation.

Reply:
Actually, this is very much NPM Targets related. It shouldn't be in NodeJs Extensions' documentation.

Reply:
So could you please add docs about it then?

Reply:
OK.

Reply:
I've adjusted the documentation and pushed the changes.